packages/ui/src/utils/groupMessages.ts

```diff
@@ -0,0 +1,51 @@
+// Internal Style Dictionary methods
```

Comment:
Copied from style-dictionary@3.9.1 which is specified in the ui package.json, no changes made